custom_components/weatherxm/translations/es.json

```diff
@@ -0,0 +1,34 @@
+{
+  "config": {
+      "abort": {
+          "already_configured": "El dispositivo ya está configurado"
+      },
+      "error": {
+          "cannot_connect": "Error al conectar",
+          "invalid_auth": "Autenticación inválida",
+          "unknown": "Error inesperado"
+      },
+      "step": {
+          "user": {
+            "title": "Opciones de WeatherXM",
+            "description": "Defina los detalles de su cuenta WeatherXM.",
```

Comment:
Let's make it a bit more colloquial:

`Define los detalles de tu cuenta WeatherXM`